src/main/java/co/inventorsoft/springresttask/controller/EmailDeliveryController.java

```diff
@@ -0,0 +1,56 @@
+package co.inventorsoft.springresttask.controller;
+
+
+import co.inventorsoft.springresttask.controller.dto.EmailDto;
+import co.inventorsoft.springresttask.service.EmailDeliveryService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/email")
+@RequiredArgsConstructor
+@Slf4j
+public class EmailDeliveryController {
+
+    private final EmailDeliveryService emailDeliveryService;
+
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
HTTP status OK is returned by default, this annotation is redundant here and in other methods below